main.cu

```diff
@@ -12,6 +18,66 @@ inline void gpuAssert(cudaError_t code, const char *file, int line, bool abort=t
    }
 }
 
+
+//=========================================
+
+//Naive Implementation of Euclidean_distance_matrix (BruteForce)
+extern "C" void matrix_Euclidean_distance_same_block_cpu(double * a, double * b, double * c, int * n, int * m, int * m_b) {

```

Comment:
Could you move this file to main.cpp and compile separately?

Reply:
ok... Should I add that to Make as well?